### ISSUE TITLE
Accessibility: Usability: Keyboard: The dropdown buttons are not visible for the list item links 

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#43]
#A11yUsable
#Wipro
#vscode-website
#FireFox

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Repro Steps:**
1. Open [https://code.visualstudio.com/docs](https://code.visualstudio.com/docs) in Edge.
2. Navigate to the LHS pane available beside the "Get Started" header.
3. Observe the dropdown buttons for the overview links [refer screenshot]
**_Note: Same issue observed for all  links having Overview contents at the LHS pane_**

**Actual Result:**
The dropdown buttons are not visible for the list items(links) at LHS pane beside the "Getting Started" header of the page while navigating with keyboard.
Note: The dropdown buttons are visible when we hover mouse over the links [refer screenshot]

**Expected Result:**
While navigating with keyboard the dropdown buttons should be visible for the links ,so that user should be able access the contents under the dropdown.

**User Impact:**
Users with visual and/or motor impairments who rely on keyboard may not be able to access the contents under the dropdown.

**Screenshots:**
![dropdown button is not visible while we navigate with keyboard](https://cloud.githubusercontent.com/assets/24825441/21715277/960793e8-d429-11e6-9571-cd9024b60f4b.png)
![dropdown button is visible with mouse](https://cloud.githubusercontent.com/assets/24825441/21715278/960805f8-d429-11e6-81a3-5990d1dfb995.png)

**Comment by chrisdias** [comment]
It is possible to press `Enter` to expand the navigation tree on the left hand side. Narrator also announces that the top level section (e.g. SETUP) is expanded or collapsed.

**Comment by chrisdias** [comment]
Seems like it has to be expanded once before Narrator announces the expanded/collapsed state... should look into this more.

**Comment by chrisdias** [comment]
fixed with https://github.com/Microsoft/vscode-docs/commit/f7da14520eb5f42c326daf07cdb3615d812cc8dd
